### ISSUE TITLE
Prisma Migrate Crash on change required foreign key to not-required

### ISSUE THREAD

**Issue by Bariyard** [prisma/migrate#626]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v12.18.2           | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs:213:26] AlterColumn on unknown column

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```
from 
roleId     String   @default(cuid())
Role       Role     @relation(fields: [roleId], references: [id])
 
to
roleId     String?   @default(cuid())
Role       Role?     @relation(fields: [roleId], references: [id])

**Comment by tomhoule** [comment]
This is a known issue in 2.10.0, thanks for reporting. We are working on this.

**Comment by tomhoule** [comment]
We worked on this problem and just released a fix in 2.10.1. Can you confirm that this does not happen anymore with the new version?

**Comment by tomhoule** [comment]
Other users confirmed the problem is gone for them in 2.10.1, so I'll close this as duplicate.
